frontend/simple/views/Contributions.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <main>
+    <div class="section is-hero">
+      <i18n tag="h1" class="title is-1 is-marginless">
+        Contributions
+      </i18n>
+      <i18n tag="p">See everyone’s contributions to the group and manage yours.</i18n>
+    </div>
+
+    <div class="section columns is-desktop is-multiline c-grid">
+      <section class="column">
+        <i18n tag="h2" class="title is-3">Receiving</i18n>
+
+        <ul class="c-ul">
+          <contribution
+            v-for="contribution, index in receiving"
+            :variant="isMonetary(contribution.type) ? 'editable' : undefined"
+            v-on="isMonetary(contribution.type) ? { 'control-click': showReceivingSettings } : {}"
```

Comment:
We need to avoid the use of 'control-click' for a couple reasons: (1) it can be confusing for some users, especially Mac users, who are not used to control-clicking things, and (2) it doesn't exist on mobile

Reply:
control-click doesn't have nothing to do with <kbd>ctrl</kbd> key... it means the dev controls the click on the contribution and doesn't trigger the input, yah... horrible name. I'll find a better name and let you know about it